grafast/grafserv/src/servers/lambda/index.ts

```diff
@@ -0,0 +1,132 @@
+import type {
+  APIGatewayEvent,
+  APIGatewayProxyEvent,
+  APIGatewayProxyResult,
+  Context as LambdaContext,
+} from "aws-lambda";
+import type {
+  GrafservConfig,
+  RequestDigest} from "grafserv";
+import {
+  GrafservBase,
+  processHeaders
+} from "grafserv";
```

Comment:
Self-imports can cause issues for various pieces of tooling (that haven't been implemented to fully support Node's resolution); please use relative paths instead

```suggestion
import type { GrafservConfig, RequestDigest } from "../../interfaces.js";
import { GrafservBase } from "../../core/base.js";
import { processHeaders } from "../../utils.js";
```

Reply:
Ya, leftover from developing this inside my own package before moving it out